src/installer.ts

```diff
@@ -107,7 +107,7 @@ export class DotnetVersionResolver {
   }
 
   static DotNetCoreIndexUrl: string =
-    'https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json';

```

Comment:
Is it the same ? Does it update at the same rate ?

Reply:
We had an internal discussion with @jamshedd. He has confirmed that this file is updated as soon as .Net versions updates are available, so we are good to go.